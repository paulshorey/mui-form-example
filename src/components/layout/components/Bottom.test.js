import React from 'react';
import renderer from 'react-test-renderer';

import Bottom from './Bottom';

it('renders correctly', () => {
	const tree = renderer.create(<Bottom />).toJSON();
	expect(tree).toMatchSnapshot();
});
