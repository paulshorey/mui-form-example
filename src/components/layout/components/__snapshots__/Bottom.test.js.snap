// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Bottom sc-bdVaJa idObzk"
>
  <span
    className="logo"
  >
    <img
      alt="Legrand"
      src="/branding/logo/legrand_white.svg"
    />
  </span>
  <span
    className="copyright"
  >
    <span
      className="desktop_only"
    >
      This app is non-functional without the Mui hardware. 
    </span>
    <a
      className="color_success"
      href="/XWC1001/examples/validation"
      style={
        Object {
          "fontWeight": "bold",
        }
      }
    >
      CLICK HERE TO DEMO the UI Â»
    </a>
  </span>
</div>
`;
